tbx/project_styleguide/templates/patterns/organisms/blog-listing/blog-listing.html

```diff
@@ -1,3 +0,0 @@
-{% for post in blog_posts %}
```

Comment:
@chris-lawton @engineervix I think this file may have been useful for the js implementation. We don't need it now, and the equivalent ones for work-listing and latest-posts can probably be removed in due course.